site/en/guide/variable.ipynb

```diff
@@ -415,7 +414,7 @@
   "metadata": {
     "colab": {
       "collapsed_sections": [],
-      "name": "intro_to_variables.ipynb",
+      "name": "variable.ipynb",
```

Comment:
How was this changed?

Reply:
It sets the colab "name" to the file-name.

Reply:
How or why? This is working as intended---it matches the filename

Reply:
Why does it do that?

What if I want the name to be what I set it to?

Reply:
Why? `intro_to_variables.ipynb` does not exist, someone didn't update it when they moved the file. This isn't the title.